### PR TITLE
Run report consolidation

### PR DESCRIPTION
**Pull request by morgantaschuk**
I consolidated the code that @apmasell wrote at SeqWare::Report with the code that @lheisler wrote at GSI::report into a new GSI::RunReport, hoping that I haven't broken anything significant.

Highlights:
- Skinned with GSI's bootstrap header and footer
- new option "-n" for not collapsing yield and coverage using reads per start point
- TSV file writing does not pick apart HTML
- pod documentation
- moved the runreport-specific CSS into a local file

### CODE REVIEW

File: jsonToGenericRunReport.pl [hunk 1]
```diff
@@ -83,77 +93,72 @@
...
 
 exit;
 
+sub get_custom_head {
+	use File::Slurp;
+	my $htmlHead="<style type=\"text/css\">\n";
+	$htmlHead.= read_file($scriptPath."/css/runreport.css");
+	$htmlHead.="</style>\n";
+	return $htmlHead;
+}
```
Comment (lheisler) [inline jsonToGenericRunReport.pl]: Should sub get_custom_head go into GSI::RunReport ?

**Review by lheisler — APPROVED** [review]

File: jsonToGenericRunReport.pl (hunk 1 above)
Comment (morgantaschuk) [reply jsonToGenericRunReport.pl]: Yep, you're right. I moved it.

**Review by apmasell — APPROVED** [review]
I might be nice to run all the code through perltidy.

**Comment by morgantaschuk** [comment]
Oh right, I was going to do that (run through perltidy). I might open a new PR though.

**Pull request merged**